### DISCUSSION TITLE
Ícone Feedback Screen Mística

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1412] [Proposals]
### Goal

Ajustar ícone no código como está no Figma.

### Context

Encontramos uma diferença no tamanho do ícone apresenta em código (64px) e no Figma (48px).

<img width="1447" alt="Captura de Tela 2023-08-18 às 15 47 10" src="https://github.com/Telefonica/mistica-design/assets/127137234/fa65352a-bbc3-44d1-a299-6ac2982dce58">

<img width="551" alt="Captura de Tela 2023-08-18 às 15 43 56" src="https://github.com/Telefonica/mistica-design/assets/127137234/6e0b3373-2634-4ab6-abdc-6cff2c6f0912">

### Impact

Ajustar o ícone em código como está no Figma faria com que em ambos os lugares refletissem a mesmo padrão do componente.

### Discussion Questions

ND.

**Comment by yceballost** [comment]
This is work in progress https://github.com/Telefonica/mistica-web/pull/852

Please dont open discussion for report bugs. You can create an issue to report bugs like this.

**Reply by annefranca** [reply]
@yceballost 

Obrigada!

Qual seria o caminho pra relatar bugs?

**Reply by yceballost** [reply]
1. Go to [issues](https://github.com/Telefonica/mistica-design/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)
2. Click on "New issue" 
3. Choose type of issue (Bug report in this case)
